agent/agents/mysql/perfschema/perfschema.go

```diff
@@ -411,7 +411,7 @@ func makeBuckets(current, prev summaryMap, l *logrus.Entry, maxQueryLength int32
 		mb := &agentv1.MetricsBucket{
 			Common: &agentv1.MetricsBucket_Common{
 				Schema:                 pointer.GetString(currentESS.SchemaName), // TODO can it be NULL?
-				Queryid:                *currentESS.Digest,
+				Queryid:                queryID,
```

Comment:
we should send query ID without prefix to qan-api, we need prefix only locally to not loose data.